src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonomyType.php

```diff
@@ -32,6 +33,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'form_type' => 'sylius_taxonomy_translation',
                 'label'    => 'sylius.form.taxonomy.name'
             ))
+            ->addEventSubscriber(new SetRootTaxonNameFormListener())
```

Comment:
Would that be something like adding:
```
        <service id="sylius.form.subscriber.build_taxonomy_form" class="Sylius\Bundle\TaxonomyBundle\Form\EventListener\BuildTaxonomyFormListener">
            <tag name="form.subscriber" alias="sylius_taxonomy_form" />
        </service>```
to src/Sylius/Bundle/TaxonomyBundle/Resources/config/services.xml ?
If so, how would I then reference the service in src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonomyType.php ?